intellij/src/saros/intellij/ui/tree/ContactPopMenu.java

```diff
@@ -26,33 +29,67 @@
 import saros.repackaged.picocontainer.annotations.Inject;
 
 /**
- * Contact pop-up menu for selecting a project to share. Opened when right-clicking on a contact.
+ * Contact pop-up menu for selecting a project and module to share. Opened when right-clicking on a
+ * contact.
  */
 class ContactPopMenu extends JPopupMenu {
 
   private static final Logger LOG = Logger.getLogger(ContactPopMenu.class);
 
   @Inject private IWorkspace workspace;
 
-  @Inject private Project project;
-
   private final ContactTreeRootNode.ContactInfo contactInfo;
 
   ContactPopMenu(ContactTreeRootNode.ContactInfo contactInfo) {
     this.contactInfo = contactInfo;
 
-    if (workspace == null || project == null) {
-      SarosPluginContext.initComponent(this);
+    SarosPluginContext.initComponent(this);
+
+    JMenu menuShareProject = new JMenu("Work together on...");
```

Comment:
Why do you use a constant string here instead of storing the menu title in messages.properties?

Reply:
It was implemented this way from the start and I just forgot about it. Fixed.